src/garage/sampler/is_sampler.py

```diff
@@ -1,23 +1,25 @@
+"""ISSampler."""
```

Comment:
Please don't just use the class name as the module comment. This should describe what's inside the module.